reader-osm/src/test/java/com/graphhopper/routing/RoutingAlgorithmWithOSMIT.java

```diff
@@ -570,7 +529,7 @@ Graph runAlgo(TestAlgoCollector testCollector, String osmFile,
             throw new RuntimeException("cannot handle " + algoEntry.toString() + ", for " + tmpOneRun
                     + ", file " + osmFile + ", " + ex.getMessage(), ex);
         } finally {
-            // Helper.removeDir(new File(graphFile));
+            Helper.removeDir(new File(graphFile));
```

Comment:
Now that we apply different CH preparations (i.e. due to different weightings) we need to get rid of the graph file ... to avoid reimport for a new test file with the same area we could attach the weighting to the graph folder.